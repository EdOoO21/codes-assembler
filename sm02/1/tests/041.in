-320 -528 -660 847 871 -211 183 835 413 -420 132 -434 -215 744 498 563 180 -981 599 -490 354 718 -963 -101 943 -683 -694 481 -347 -657 -473 340 415 271 195 -879 -755 606 203 -933 -115 775 -423 722 504 -558 -852 487
