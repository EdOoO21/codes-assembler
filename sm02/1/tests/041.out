487
-852
-558
504
722
-423
775
-115
-933
203
606
-755
-879
195
271
415
340
-473
-657
-347
481
-694
-683
943
-101
-963
718
354
-490
599
-981
180
563
498
744
-215
-434
132
-420
413
835
183
-211
871
847
-660
-528
-320
